CHANGELOG.md

```diff
@@ -9,7 +9,12 @@ Every entry has a category for which we use the following visual abbreviations:
 - ⚠️ change
 - 🐞 bugfix
 
-<!-- ## Unreleased -->
+## Unreleased
+
+- ⚠️ On Linux, VAST no longer looks for the `share/vast` directory relative to
+  the location of the `vast` binary, because users preferred to always use
+  `/etc/vast` instead. On macOS and FreeBSD, both paths are still looked at.
+  [#1036](https://github.com/tenzir/vast/pull/1036)
```

Comment:
```suggestion
- ⚠️ On Linux, VAST no longer looks for the `share/vast` directory relative to
  the location of the `vast` binary, because Linux distributions expect that a system-wide configuration in `/etc` takes precedence over one in `/usr/local/etc`. On macOS and FreeBSD, both paths are still looked at.
  [#1036](https://github.com/tenzir/vast/pull/1036)
```

Reply:
I gave it another shot to make sure I understood this correctly. Does that read fine?

Reply:
That is mostly correct, except that we're not looking at `/usr/local`, it's just that it is the most common install prefix in practice.

Reply:
Okay, so `/usr/local` -> `PREFIX` assuming that PREFIX is not `/`.